src/Common/Admin/Help_Hub/Hub.php

```diff
@@ -126,23 +161,59 @@ public function register_hooks(): void {
 		add_filter( 'admin_body_class', [ $this, 'add_help_page_body_class' ] );
 	}
 
+	/**
+	 * Registers the hidden admin page for the Help Hub.
+	 *
+	 * @since 6.3.2
+	 *
+	 * @return void
+	 */
+	protected function register_hidden_page(): void {
+		add_action(
+			'admin_menu',
+			function () {
+				add_submenu_page(
+					null, // Make the page hidden.
+					__( 'Help Hub', 'tribe-common' ),
+					__( 'Help Hub', 'tribe-common' ),
+					'manage_options',
+					self::IFRAME_PAGE_SLUG,
+					[ $this, 'render' ]
+				);
+			},
+			999
+		);
```

Comment:
@redscar Why the anonymous function? Can there be a case in the future where we want to replace this by remove_action? We won't be able to remove it then.